tox.ini

```diff
@@ -36,7 +36,7 @@ setenv =
     SUPERSET_CONFIG = tests.superset_test_config
     SUPERSET_HOME = {envtmpdir}
     py27-mysql: SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://mysqluser:mysqluserpassword@localhost/superset?charset=utf8
-    py34-mysql: SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://mysqluser:mysqluserpassword@localhost/superset
+    py{34,36}-mysql: SUPERSET__SQLALCHEMY_DATABASE_URI = mysql://mysqluser:mysqluserpassword@localhost/superset
```

Comment:
Side note, we may be able to drop `3.4` support soon. While `2.7` should have a long tail of support, I don't think it's as needed with `3.4`

Reply:
Agreed. This PR just adds the missing Python 3.6 version which I was using for testing locally. Note we've kind of already dropped 3.4 per [here](https://github.com/apache/incubator-superset/blob/master/setup.py). I'll create a new PR to remove reference to it in Tox.